src/index.js

```diff
@@ -44,7 +43,7 @@ function Multiaddr (addr) {
   } else {
     throw new Error('addr must be a string, Buffer, or another Multiaddr')
   }
-}
+}, { className: 'Multiaddr', symbolName: '@multiformats/js-multiaddr/multiaddr' })
```

Comment:
@fsdiogo instead of adding it at the top (which makes it look like a 👽 function declaration for folks non familiar with the whole situation), do the withIs.proto a separate statement before the module exports.